<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowNovoVeiculo</class>
 <widget class="QMainWindow" name="MainWindowNovoVeiculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Novo veiculo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Modelo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>33</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Placa:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>100</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kilometragem:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor de Compra:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Situação:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMarcaVeiculo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>11</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditModeloVeiculo">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>40</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPlacaVeiculo">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>70</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKmVeiculo">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>100</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValordeCompra">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>129</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Km</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>130</y>
      <width>17</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R$:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSalvarVeiculo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancelarVeiculo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDisponivel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Disponivel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancelarVeiculo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowNovoVeiculo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
